src/ftp-deploy.spec.js

```diff
@@ -66,8 +64,7 @@ describe("ftp-deploy.spec: deploy tests", () => {
             .then(() => {
                 // Should reject if file does not exist
                 return statP(remoteDir + "/test-inside-root.txt");
-            })
```

Comment:
what is the intention here?

Reply:
removed this, it was an unnecessary change